app/src/main/java/org/mozilla/tv/firefox/ext/Js.kt

```diff
@@ -54,6 +54,7 @@ var _firefoxTV_isPlaybackStateObserverLoaded;
     function onDOMChangedForVideos() {
         addPlaybackStateListeners();
         syncPlaybackState();
+        adjustVideoCSS();
```

Comment:
It's _possible_ adjusting the video CSS will change the playback state: perhaps we should do this first, before the other functions.